.travis.yml

```diff
@@ -3,9 +3,10 @@ python:
   - "2.7"
   - "3.4"
 env:
-  - DJANGO_VERSION=1.6
```

Comment:
I'm not quite sure why coverage decreases so much, but perhaps it's because of removing this part and then south migrations don't get run?